src/mumble/UserView.cpp

```diff
@@ -107,18 +109,26 @@ bool UserDelegate::helpEvent(QHelpEvent *evt, QAbstractItemView *view, const QSt
 	return QStyledItemDelegate::helpEvent(evt, view, option, index);
 }
 
-UserView::UserView(QWidget *p) : QTreeView(p) {
+UserView::UserView(QWidget *p) : QTreeView(p), m_userDelegate(make_qt_unique< UserDelegate >(this)) {
 	// Calculate the icon size for status icons based on font size
 	// This should automaticially adjust size when the user has
 	// display scaling enabled
-	m_flagTotalDimension  = QFontMetrics(p->font()).height();
-	int flagIconPadding   = 1;
-	int flagIconDimension = m_flagTotalDimension - (4 * flagIconPadding);
-	setItemDelegate(new UserDelegate(this, m_flagTotalDimension, flagIconPadding, flagIconDimension));
+	adjustFlag();
+	setItemDelegate(m_userDelegate.get());
+
+	QTimer::singleShot(0, [this]() { adjustFlag(); });
```

Comment:
```suggestion
	// Because in Qt fonts take some time to initialize properly, we have to delay the call
	// to adjustFlag a bit in order to give the fonts the necessary time (so we can read out
	// the actual font details).
	QTimer::singleShot(0, [this]() { adjustFlag(); });
```